### PR TITLE
Update for new register backend image for Data Archival to cloud

### PR DESCRIPTION
**Pull request by kumarashit**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
/kind new feature
/kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:
Add update for data archival changes and support metrics for multi-cloid.
**Note** Support metrics need to be updated for feature list

**Which issue(s) this PR fixes**:
Fixes # 151

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rajat-soda** [comment]
LGTM

**Comment by PravinRanjan10** [comment]
LGTM

File: content/support-matrix/multi-cloud/_index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: multi-cloud
+menuTitle: multi-cloud
```
Comment (anvithks) [inline content/support-matrix/multi-cloud/_index.md:3]: multi-cloud -> Multi-cloud

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This guide shows how to archive objects into Cloud object storage and restore th
 To refer the API specs, please check [Cloud Object Archival API Specs](https://github.com/sodafoundation/multi-cloud/blob/master/openapi-spec/swagger.yaml)
 
 {{% notice note %}}
- Currently this feature is available only for AWS cloud
+ Currently this feature is available only for AWS, Azure and GCP cloud as backend and SONY-ODA as on-premise Cold storage
 {{% /notice %}}
 
 ## Installing SODA multi-cloud
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:25]: Could you please check the links at the Pre-requisite section below: they are not appearing correctly.
I believe the correct linking would be [How to register a Cloud Backend](/#how-to-register-a-cloud-backend)
`[How to register a Cloud Backend](/#how-to-register-a-cloud-backend)`
image

**Review by NajmudheenCT — DISMISSED** [review]
LGTM

File: content/support-matrix/multi-cloud/_index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+description: "Matrix data for supported cloud vendors and features"
+weight: 15
+disableToc: false
+tags: ["support matrix", "multi-cloud", "cloud vendors"] 
+---
+This section provides the metrics about the supported cloud providers and the features for SODA multi-cloud.
```
Comment (anvithks) [inline content/support-matrix/multi-cloud/_index.md:9]: metrics -> matrix.

File: content/support-matrix/nbp/_index.md [hunk 4]
```diff
@@ -2,10 +2,10 @@
 title: NBP
 menuTitle: NBP
 description: "All user guides for the North Bound Plugin(NBP) project can be found here"
-weight: 10
+weight: 15
 disableToc: false
-tags: ["user guide", "delfin", "dashboard"] 
+tags: ["user guide", "nbp", "dashboard"]
```
Comment (anvithks) [inline content/support-matrix/nbp/_index.md:7]: could you please remove the dashboard tag from here.

**Review by anvithks — CHANGES_REQUESTED** [review]
@kumarashit Please check some link issues and formatting.

One thing that is missing is to add a note in archival and retrieval regarding archived objects in  GCP not requiring `Restore` as they are implicitly rehydrated / restored when downloaded.

Similarly a note about the Azure Hot/Cool tier and only restoring to Standard priority. (This may not be needed here.)

**Comment by kumarashit** [comment]
> @kumarashit Please check some link issues and formatting.
> 
> One thing that is missing is to add a note in archival and retrieval regarding archived objects in GCP not requiring `Restore` as they are implicitly rehydrated / restored when downloaded.
> 
> Similarly a note about the Azure Hot/Cool tier and only restoring to Standard priority. (This may not be needed here.)

Mentioning this may require more details. But already architecture-analysis link s provided

**Comment by anvithks** [comment]
> > @kumarashit Please check some link issues and formatting.
> > One thing that is missing is to add a note in archival and retrieval regarding archived objects in GCP not requiring `Restore` as they are implicitly rehydrated / restored when downloaded.
> > Similarly a note about the Azure Hot/Cool tier and only restoring to Standard priority. (This may not be needed here.)
> 
> Mentioning this may require more details. But already architecture-analysis link s provided

> > One thing that is missing is to add a note in archival and retrieval regarding archived objects in GCP not requiring `Restore` as they are implicitly rehydrated / restored when downloaded.

I think this may need to be added at the least since there is a difference in the way it operates on the frontend.

File: content/support-matrix/nbp/_index.md (hunk 4 above)
Comment (kumarashit) [reply content/support-matrix/nbp/_index.md]: Done

File: content/support-matrix/multi-cloud/_index.md (hunk 3 above)
Comment (kumarashit) [reply content/support-matrix/multi-cloud/_index.md]: Done

File: content/guides/user-guides/multi-cloud/object-archival-restore/_index.md (hunk 2 above)
Comment (kumarashit) [reply content/guides/user-guides/multi-cloud/object-archival-restore/_index.md:25]: Corrected. Thanks for checking

**Review by anvithks — APPROVED** [review]
LGTM

**Review by NajmudheenCT — APPROVED** [review]
LGTM

**Pull request merged**